### ISSUE TITLE
Breaking change in minor version.

### ISSUE THREAD

**Issue by jsirex** [chef/mixlib-log#39]
From Gemfile.lock:
```
    chef (12.22.3)
      mixlib-authentication (~> 1.4)
      mixlib-log (~> 1.3)
```

mixlib-authentication uses `logger.trace`, but it is available in mixlib-log 2.x, isn't it?

Anyway, mixlib-authentiation 1.4.2 works.
I believe the easiest way is to add trace here in 1.x version.

There are also no explicit dependines from mixlib-authentication to mixlib-log.
I blame this commit: https://github.com/chef/mixlib-authentication/commit/9384e69b748deced867cb544095bb22d8f60e83d

**Comment by jsirex** [comment]
Don't you think that better solution just release mixlib-log 1.7.2 with `trace` method added?

**Comment by thommay** [comment]
No. We don't support or recommend installing chef from gems, so this isn't really a problem for many of our users. We don't want to rework logging levels in chef 12 or 13, so we'll just ensure that the right versions of mixlib-authentication are installed in the right place.
